modules/augment/include/opencv2/augment/gaussianblur.hpp

```diff
@@ -0,0 +1,51 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+
+#ifndef OPENCV_AUGMENT_GAUSSIANBLUR_HPP
+#define OPENCV_AUGMENT_GAUSSIANBLUR_HPP
+#include <opencv2/augment/transform.hpp>
+
+namespace cv { namespace augment {
+
+class CV_EXPORTS_W GaussianBlur : public Transform
+{
+public:
+    /* @brief Constructor to initialize the gaussian blur transformation with a range of kernel sizes and range of sigma values
+       @param minKernelSize the min size of gaussian blur kernel
+       @param maxKernelSize the max size of gaussian blur kernel
+       @param minKernelSize the min sigma value of the gaussian blur kernel
+       @param maxKernelSize the max sigma value of the gaussian blur kernel
+    */
+    CV_WRAP GaussianBlur(int minKernelSize, int maxKernelSize, float minSigma, float maxSigma);
```

Comment:
Initialization parameters must as close to the origin method parameters as it's possible:

```cpp
Size ksize,
double sigmaX,
double sigmaY = 0
```
So I think we need to have the following constructors:
1) Size ksize, double sigmaX, double sigmaY = 0
2) int ksize, double sigma = 0 (for both dimensions)
3) Size min_kize, Size max_ksize, double minSigmaX, double maxSigmaX, double minSigmaY = 0, double maxSigmaY = 0
4) int min_ksize, int max_ksize, double minSigma = 0, double maxSigma = 0